### DISCUSSION TITLE
Device configuration file problem

### DISCUSSION THREAD

**Discussion by HolBaum5** [zwave-js/zwave-js#6667] [Q&A] [answered]
Hello, I have a Z-TRV-V01 thermostatic valve whose config is not (yet) in the database. 
I have put the config as described here (https://community.home-assistant.io/t/how-to-manualy-supply-z-wave-js-config-file-from-the-db/424393/9) to storage/config.
![grafik](https://github.com/zwave-js/node-zwave-js/assets/79714761/cf6c7005-46a1-493b-b611-f0e26763c621)
 
I have taken the config file from here https://github.com/zwave-js/node-zwave-js/pull/6542

But unfortunately I don't see any change in my overview - the device is still unknown. What could be the reason?
![grafik](https://github.com/zwave-js/node-zwave-js/assets/79714761/ff1adb63-f201-4e22-bad3-df1a18e4949e)

**Comment by HolBaum5** [comment]
Sorry wrong project can be deleted

**Reply by AlCalzone** [reply, marked as answer]
You need to restart Z-Wave JS UI and re-interview the device.
